src/rdfTypes/GeometryInfoHelpersImpl.h

```diff
@@ -255,27 +255,28 @@ enum class AnyGeometryMember : uint8_t {
 
 // Helper to convert the dynamic container `AnyGeometry` to the `ParsedWkt`
 // variant type
-inline ParsedWkt anyGeometryToParsedWkt(AnyGeometry<double> geom) {
+template <typename Visitor>
+inline auto visitAnyGeometry(Visitor visitor, const DAnyGeometry& geom) {
```

Comment:
That is nice, do you also have cases where you want non-constness, or even to be able to move out of the `DAnyGeometry`, then we can adapt this.


PS: For your PRs on monday: Just base them off this PR, and later merge in once this becomse part of the master.